sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -58,11 +58,13 @@ To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
 
 ##### Example
-```java readme-sample-createFarmersBaseClient
-FarmersBaseClient client = new FarmBeatsClientBuilder()
-        .endpoint("https://<farmbeats resource name>.farmbeats-dogfood.azure.net")
-        .credential(new DefaultAzureCredentialBuilder().build())
-        .buildFarmersBaseClient();
+
+```java readme-sample-createPartiesClient
+PartiesClientBuilder builder = new PartiesClientBuilder()
+    .host("https://<farmbeats-endpoint>.farmbeats.azure.net")
+    .httpClient(HttpClient.createDefault());
+PartiesClient partiesClient = builder.buildClient();
+Response<BinaryData> response = partiesClient.getWithResponse("<party-id>", new RequestOptions());
 ```
 
 ## Key concepts
```

Comment:
please double check key concepts still applies. e.g. farmer -> partner

Reply:
Yup, API itself has been updated and it's backward compatible.

Reply:
I mean these
https://github.com/Azure/azure-sdk-for-java/blob/7646fd403bd0980426bd1de9186f26581f4bd55c/sdk/agrifood/azure-verticals-agrifood-farming/README.md#key-concepts

If you change farmer to partner in API, why tell user about "farmer" in key concepts?

Reply:
Fixed links, updated readme with some samples as well.